crypter/shield-crypt/shield-crypt.go

```diff
@@ -0,0 +1,82 @@
+package main
+
+import (
+	"crypto/cipher"
+	"encoding/hex"
+	"encoding/json"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/jhunt/go-cli"
+	"github.com/starkandwayne/shield/crypter"
+)
+
+type CLIOpts struct {
+	EncryptShort bool `cli:"-e"`
+	DecryptShort bool `cli:"-d"`
+	EncryptFull  bool `cli:"--encrypt"`
+	DecryptFull  bool `cli:"--decrypt"`
+}
+
+func main() {
+	var encStream, decStream cipher.Stream
+	var data struct {
+		EncryptionKey  string `json:"enc_key"`
+		EncryptionIV   string `json:"enc_iv"`
+		EncryptionType string `json:"enc_type"`
+	}
```

Comment:
For clarity, rename the structure to `crypt` and drop the `Encryption` prefix on all the members.  This let's us deal with `crypt.Key`, `crypt.IV`, etc.